src/_common/content/components/media-item/media-item.vue

```diff
@@ -29,7 +33,7 @@
 					:media-item="mediaItem"
 					radius="lg"
 				>
-					<template v-if="isHydrated">
+					<template v-if="mediaItem">
```

Comment:
I think @hworld and I thought it would be best to not have the getter, just so the template isn't giving us incorrect warnings. I did remove the `!!` though, as that was unneeded.